src/Messages.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * on-the-wire JSON-stringified representations of the message payloads
+ */
+
+import { EventEnvelope, ResponseEnvelope } from './Envelope'
+import { MessageType } from './MessageType'
+
+type Response<TType extends MessageType, TPayload> = ResponseEnvelope & {
+  type: TType
+  payload: TPayload
+}
+
+type Event<TType extends MessageType, TPayload> = EventEnvelope & {
+  type: TType
+  payload: TPayload
+}
+
+export interface ContextPayload {
+  member?: string
```

Comment:
this is the member id?

Reply:
yes - in our REST APIs we always say `member` not `member_id`, which is why I went with this.